bin/node-pg-migrate.ts

```diff
@@ -255,8 +272,10 @@ let CHECK_ORDER = argv[checkOrderArg];
 let VERBOSE = argv[verboseArg];
 let DECAMELIZE = argv[decamelizeArg];
 let tsconfigPath = argv[tsconfigArg];
+let useTsNode = argv[tsNodeArg];
+let useTsx = argv[tsxArg];
 
-function readTsconfig() {
+function readTsconfig(): void {
   if (tsconfigPath) {
```

Comment:
It would be nice to be able to use tsx without specifying the `tsconfigPath` I guess.